src/main/java/jenkins/test/RunMatchers.java

```diff
@@ -116,4 +123,16 @@ public void describeTo(Description description) {
             description.appendText("log containing ").appendValue(message);
         }
     }
+
+    private static class CompletedRunMatcher extends TypeSafeMatcher<Run<?, ?>> {
+        @Override
+        protected boolean matchesSafely(Run<?, ?> run) {
+            return !run.isLogUpdated();
```

Comment:
So I think https://github.com/jenkinsci/jenkins-test-harness/blob/f2a982bb910fa3fc2998586276a58e5042530e1d/src/main/java/org/jvnet/hudson/test/JenkinsRule.java#L1632 could be deleted and that method could also suggest using this matcher.